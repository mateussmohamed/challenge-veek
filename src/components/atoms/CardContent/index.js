import styled from 'styled-components'

const CardContent = styled.section`
  width: 100%;
  padding: 20px 20px 0 20px;
  color: #000000;
  font-family: Lato, sans-serif;
  font-size: 16px;
  font-weight: 400;
`

export default CardContent
