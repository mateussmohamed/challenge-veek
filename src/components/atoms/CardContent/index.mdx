---
name: CardContent
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import CardContent from './index'

# CardContent
<PropsTable of={CardContent} />

<Playground>
  <CardContent />
</Playground>
