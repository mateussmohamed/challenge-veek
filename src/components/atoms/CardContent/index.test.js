import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import CardContent from './index'

describe('[<CardContent />]', () => {
  const wrap = (props = {}) => shallow(<CardContent {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a container!' })

    expect(wrapper).toMatchSnapshot()
  })
})
