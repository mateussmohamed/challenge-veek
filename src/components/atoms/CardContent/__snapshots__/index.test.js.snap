// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<CardContent />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "CardContent-mia19x-0",
        "isStatic": true,
        "rules": Array [
          "width:100%;padding:20px 20px 0 20px;color:#000000;font-family:Lato,sans-serif;font-size:16px;font-weight:400;",
        ],
      },
      "displayName": "CardContent",
      "render": [Function],
      "styledComponentId": "CardContent-mia19x-0",
      "target": "section",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a container!
</StyledComponent>
`;
